core/src/main/java/com/graphhopper/routing/util/spatialrules/countries/GreatBritainSpatialRule.java

```diff
@@ -0,0 +1,56 @@
+/*
+ *  Licensed to GraphHopper GmbH under one or more contributor
+ *  license agreements. See the NOTICE file distributed with this work for
+ *  additional information regarding copyright ownership.
+ *
+ *  GraphHopper GmbH licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except in
+ *  compliance with the License. You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package com.graphhopper.routing.util.spatialrules.countries;
+
+import com.graphhopper.routing.util.spatialrules.AccessValue;
+import com.graphhopper.routing.util.spatialrules.DefaultSpatialRule;
+import com.graphhopper.routing.util.spatialrules.TransportationMode;
+
+/**
+ * Defines the default rules for roads in Great Britain
+ *
+ * @author Robin Boldt
+ */
+public class GreatBritainSpatialRule extends DefaultSpatialRule {
+
+    @Override
+    public double getMaxSpeed(String highwayTag, double _default) {
+        // As defined in: https://wiki.openstreetmap.org/wiki/OSM_tags_for_routing/Maxspeed#Motorcar
+        switch (highwayTag) {
+            case "motorway":
+                return 113;
+            default:
+                return super.getMaxSpeed(highwayTag, _default);
+        }
+    }
+
+    @Override
+    public AccessValue getAccessValue(String highwayTag, TransportationMode transportationMode, AccessValue _default) {
```

Comment:
I think this method is a bit limited, we should think about passing the ReaderWay, this would allow us to make more complex decision also using other available tags, for example: `designation=public_bridleway`, which is a relevant tag for bridleways in the UK, but not for other countries, see [here](https://wiki.openstreetmap.org/wiki/Tag:highway=bridleway).